app/src/main/java/com/firebase/uidemo/database/ChatActivity.java

```diff
@@ -131,26 +136,50 @@ public void onAuthStateChanged(@NonNull FirebaseAuth firebaseAuth) {
     }
 
     private void attachRecyclerViewAdapter() {
-        Query lastFifty = mChatRef.limitToLast(50);
-        mAdapter = new FirebaseRecyclerAdapter<Chat, ChatHolder>(
-                Chat.class, R.layout.message, ChatHolder.class, lastFifty) {
+        mAdapter = new FirebaseIndexRecyclerAdapter<Chat, ChatHolder>(
```

Comment:
While I definitely appreciate your constant testing of the indexed bits, the purpose of `ChatActivity` is to show off the basic FBUI integration, and I think `FirebaseRecyclerAdapter` is the most popular.  People are likely to copy and paste from this class and I want them to get the least surprising thing.

Is there somewhere else we can put the indexed code?  Maybe in another Activity that's not exposed in the main chooser?